catalog/app/containers/Bucket/UploadDialog.js

```diff
@@ -758,6 +762,11 @@ export default function UploadDialog({ bucket, open, onClose }) {
           meta: getMetaValue(meta),
         },
       })
+      if (refresh) {
+        // wait for ES index to receive the new package data
+        await new Promise((resolve) => setTimeout(resolve, ES_LAG))
+        refresh()
```

Comment:
Hm... I would expect, that `refresh` reloads page or reloads some component. If it reloads data, I would use `reFetch`, or `reloadData`, or `retryFetching`

Reply:
well, it _refreshes_ the data, so i think it's ok semantically)

Reply:
Of course. Just sound my thoughts :)